package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "scripts": {
     "test": "bundlesize && jest --coverage --no-cache",
-    "build": "npm run bundle && npm run minify",
+    "build": "npm run bundle && npm run minify && cp hyperapp.js.flow dist/",
```

Comment:
I'm not sure, I think it has to be next to the file being imported, and in this case that is dist/hyperapp.js correct?  So I think dist/hyperapp.js.flow is the only default file that flow will check for otherwise you have to add the file into .flowconfig [libs]